src/components/show-edit-pod/show-edit-pod.component.js

```diff
@@ -245,6 +250,8 @@ ShowEditPod.propTypes = {
   deleteText: PropTypes.string,
   /** Can inform if the form is in a saving state (disables the save button) */
   saving: PropTypes.bool,
+  /** Title fo be displayed */
+  title: PropTypes.oneOfType([PropTypes.string, PropTypes.object]),
 };
 
 ShowEditPod.defaultProps = {
```

Comment:
`defaultProps` will soon be deprecated from functional components so can you move these defaults into the prop destructuring at the top of the component please?